docs/docs/aws.md

```diff
@@ -378,6 +378,19 @@ However, for the older versions up to 0.12.0, the logic is as follows:
 
 For more details, please refer to the [LocationProvider Configuration](custom-catalog.md#custom-location-provider-implementation) section.  
 
+### S3 Retries
+
+Workloads which encounter S3 throttling should persistently retry, with exponential backoff, to make progress while S3
+automatically scales. The default values for S3 retries are tuned such that most workloads that encounter throttling will
+make progress without failing due to retry exhaustion. For workloads with exceptionally high throughput against tables
+that S3 has not yet scaled, it may be necessary to increase the retry count further.
+
+| Property             | Default | Description                                     |
+|----------------------|---------|-------------------------------------------------|
+| s3.retry.num-retries | 32      | Number of times to retry S3 operations.         |
+| s3.retry.min-wait-ms | 500ms   | Minimum wait time to retry a S3 operation.      |
```

Comment:
the default here doesn't match what's in the code

Reply:
Update the default wait time in the docs